tests/cancun/eip1153_tstore/test_tload_calls.py

```diff
@@ -0,0 +1,114 @@
+"""
+Ethereum Transient Storage EIP Tests
+https://eips.ethereum.org/EIPS/eip-1153
+"""
+
+from typing import Dict, Union
+
+import pytest
+
+from ethereum_test_tools import (
+    Account,
+    Address,
+    Environment,
+    StateTestFiller,
+    TestAddress,
+    Transaction,
+)
+from ethereum_test_tools.vm.opcode import Bytecode
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-1153.md"
+REFERENCE_SPEC_VERSION = "2f8299df31bb8173618901a03a8366a3183479b0"
+
+
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize("call_type", [Op.CALL, Op.CALLCODE, Op.DELEGATECALL])
```

Comment:
It would be interesting to test for EOF here in the future, but at the moment it's a bit messy given how we parametrize a fork with the opcodes that the fork supports.

I'd like to work on a marker like
```python
@pytest.mark.with_all_call_opcodes
```
that performs the covariant parametrization of the opcodes that the fork supports, e.g. on Cancun and before it would be `[Op.CALL, Op.CALLCODE, Op.STATICCALL, Op.DELEGATECALL]` but on the EOF fork and after it would be `[Op.CALL, Op.CALLCODE, Op.STATICCALL, Op.DELEGATECALL, Op.EXTCALL, Op.EXTSTATICCALL, Op.EXTDELEGATECALL]`.